middlewares/authentication-check.js

```diff
@@ -0,0 +1,15 @@
+const jwt = require('jsonwebtoken');
+
+const { SECRET } = require('../config/config');
+
+exports.authenticationCheck = (req, res, next) => {
+    const token = req.headers['authorization'];
+
+    jwt.verify(token, SECRET, function(err) {
+        if (err) {
+            next(err);
```

Comment:
The error should be a 401 Unauthorized error created with our own HTTP handler (ideally with the new errno system)